### DISCUSSION TITLE
Anify shutting down its Website & API

### DISCUSSION THREAD

**Discussion by NYT92** [amvstrm/amvstrm#45] [Announcements]
We just woke up and saw the announcement that Anify will be shut down later this month. Since we rely on their API for stream & episode, it is safe to say that We don't think We would bring my multi-stream to the website because of how complicated they are. If you guys have another hosted instance of Anify, just reply to this discussion. 

![image](https://github.com/amvstrm/amvstrm/assets/53612429/af6c3cd2-5580-4758-820d-4fdf2d209234)

RIP Anify :`(

**Comment by adgsenpai** [comment]
How would this affect the API?

**Reply by NYT92** [reply]
It only affects the "episode" and "stream/multi" v2 route. I might use @consumet/extensions on the episode route but for "stream/multi", I need to learn from the code and implement it myself.
